src/coreclr/build.cmd

```diff
@@ -378,31 +381,37 @@ REM ============================================================================
 set OptDataProjectFilePath=%__ProjectDir%\src\.nuget\optdata\optdata.csproj
 if %__RestoreOptData% EQU 1 (
     echo %__MsgPrefix%Restoring the OptimizationData Package
+    set "__BinLog=%__LogsDir%\OptRestore_%__BuildOS%__%__BuildArch%__%__BuildType%.binlog"
+
     powershell -NoProfile -ExecutionPolicy ByPass -NoLogo -File "%__RepoRootDir%\eng\common\msbuild.ps1" /clp:nosummary %__ArcadeScriptArgs%^
         %OptDataProjectFilePath% /t:Restore^
         %__CommonMSBuildArgs% %__UnprocessedBuildArgs%^
-        /nodereuse:false
```

Comment:
Is there a reason we still have `/nodereuse:false` here? Generally this is only necessary when we have conflicting build task assemblies. I wouldn't expect that in our builds.

Reply:
Now that there's ALC's coming with MSBuild, we might be able to take them out as long as we ensure contributors have such a version. However, I am not sure how pervasive this this issue was before I got here; I remember the option being set in coreclr from very early on. I remember seeing cases like this: https://github.com/dotnet/runtime/issues/12834

Reply:
This should still be required, @elinor-fung I believe has more specifics on why.